cmd/soroban-cli/src/commands/global.rs

```diff
@@ -46,6 +46,22 @@ pub struct Args {
     /// Do not cache your simulations and transactions
     #[arg(long, env = "STELLAR_NO_CACHE", global = true)]
     pub no_cache: bool,
+
+    /// RPC URL for the Stellar network
+    #[arg(long, env = "STELLAR_RPC_URL")]
+    pub rpc_url: Option<String>,
+ 
+    /// Network passphrase for the Stellar network
+    #[arg(long, env = "STELLAR_NETWORK_PASSPHRASE")]
+    pub network_passphrase: Option<String>,
+ 
+    /// Network name (e.g., 'testnet', 'mainnet')
+    #[arg(long, env = "STELLAR_NETWORK")]
+    pub network: Option<String>,
+
+    /// Path to the WebAssembly file
+    #[arg(long, env = "STELLAR_WASM", global = true)]
+    pub wasm: Option<PathBuf>,
```

Comment:
This does the opposite of what this PR is trying to solve. The issue is that the subcommand of `contract info`, require `network` which doesn't make sense since they all are reading a local file. This change adds all these arguments to the global args, e.g. `stellar --network ... contract info`, which is not what we want.